client/components/gsuite/gsuite-price/index.jsx

```diff
@@ -1,30 +1,100 @@
 /**
  * External dependencies
  */
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import { abtest } from 'lib/abtest';
 import { getAnnualPrice, getMonthlyPrice } from 'lib/gsuite';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
+/**
+ * Determines whether a discount can be applied to the specified product via a sales coupon.
+ *
+ * @param {object} product - G Suite product
+ * @returns {boolean} - true if a discount can be applied, false otherwise
+ */
+const hasValidDiscount = ( product ) => {
+	if (
+		! product ||
+		! product.sale_cost ||
+		! product.sale_coupon ||
+		! product.sale_coupon.start_date ||
+		! product.sale_coupon.expires
+	) {
+		return false;
+	}
+
+	const currentTime = Date.now();
+	const startDate = new Date( product.sale_coupon.start_date );
+	const endDate = new Date( product.sale_coupon.expires );
+
+	return currentTime >= startDate && currentTime <= endDate;
+};
+
 const GSuitePrice = ( { currencyCode, product } ) => {
 	const translate = useTranslate();
 
 	const cost = product?.cost ?? null;
-	const annualPrice = cost && currencyCode ? getAnnualPrice( cost, currencyCode ) : '-';
-	const monthlyPrice = cost && currencyCode ? getMonthlyPrice( cost, currencyCode ) : '-';
+
+	const annualPrice = getAnnualPrice( cost, currencyCode );
+	const monthlyPrice = getMonthlyPrice( cost, currencyCode );
+
+	if ( abtest( 'gsuitePrice' ) === 'discount' ) {
+		const isDiscounted = hasValidDiscount( product );
+
+		return (
+			<div className="gsuite-price">
+				<h4 className="gsuite-price__monthly-price">
+					{ translate( '{{strong}}%(price)s{{/strong}} per user/month', {
+						args: {
+							price: monthlyPrice,
+						},
+						components: {
+							strong: <strong />,
+						},
+						comment: "Monthly price per user formatted with the currency (e.g. '$8.40')",
+					} ) }
+				</h4>
+
+				<h5 className={ classNames( {
+					'gsuite-price__annual-price': true,
+					'discounted': isDiscounted
+				} ) }>
+					{ translate( '%(price)s billed annually', {
+						args: {
+							price: annualPrice,
+						},
+						comment: "Annual price formatted with the currency (e.g. '$99.99')"
+					} ) }
+				</h5>
+
+				{ isDiscounted && (
+					<h6 className="gsuite-price__discounted-price">
```

Comment:
The spacing looks a little off with this `h6` and styles you have applied. We have a component you can use instead: https://wpcalypso.wordpress.com/devdocs/design/badge.

Reply:
This is a before and after with the component:

**Before**
![image](https://user-images.githubusercontent.com/6981253/69656054-d4f78300-1045-11ea-8c61-a7d379c1ad67.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/69656038-cc06b180-1045-11ea-969d-aba25e9ed879.png)